LICENSE-3rdparty.csv

```diff
@@ -213,6 +213,7 @@ core,"github.com/davecgh/go-spew/spew",ISC
 core,"github.com/dgraph-io/ristretto",Apache-2.0
 core,"github.com/dgraph-io/ristretto/z",Apache-2.0
 core,"github.com/dgryski/go-jump",MIT
+core,"github.com/dgryski/go-minhash",MIT
```

Comment:
Where do all of these new licenses come from? If they are `go-license-detector` dependencies we should add them to the exceptions list

Reply:
@albertvaka did we deal with this (indirect dependencies of Go tooling being added to the licenses list) in some way for your Go tooling PR?

Reply:
Seems like we didn't entirely, feel free to ignore this and we will deal with it in the future if it becomes a problem

Reply:
Yeah, these transitive guys are a bit of a problem to track. Thanks for bringing it up, our  3rd-party license file is a little bloated due to these tooling deps. Hopefully we can fix it in some reliable way in the future.